src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -25,16 +29,19 @@ public class SidecarInfoTest {
   record TestInputs(
       String displayName,
       String osName,
-      String osVersion,
+      String osArch,
+      String version,
       OperatingSystemType os,
       SidecarInfo.VsCode vscode,
-      String vscodeVersionPrefix
+      String vscodeVersionPrefix,
+      String userAgent
   ) {
     Map<String, String> asProperties() {
       // Construct the "system" properties from the input
       Map<String, String> props = new HashMap<>();
+      props.put(OS_ARCH_KEY, osArch);
```

Comment:
we set the osArch here, but you'll notice it always comes in as the same value in the tests. ("aarch64")